src/js/service/ImportService.js

```diff
@@ -156,7 +157,6 @@
    */
   ns.ImportService.prototype.createPiskelFromImages_ = function (images, name,
       frameSizeX, frameSizeY, smoothing) {
-    name = name || 'Imported piskel';
```

Comment:
I'd keep this, and make the change on line 162 below [to match the upstream implementation](https://github.com/piskelapp/piskel/blob/bcee24609f776212050990040ea76722ff7dbe55/src/js/service/ImportService.js#L120-L123) - we don't actually need to be customized here, AFAIK.